posthog/hogql_queries/experiments/test/test_experiment_funnel_query_runner.py

```diff
@@ -90,18 +101,259 @@ def test_query_runner(self):
+                    {"event": "purchase", "timestamp": "2020-01-03", "properties": {ff_property: "control"}},
+                ],
+                "user_control_2": [
+                    {"event": "$pageview", "timestamp": "2020-01-02", "properties": {ff_property: "control"}},
+                ],
+                "user_control_3": [
+                    {"event": "$pageview", "timestamp": "2020-01-02", "properties": {ff_property: "control"}},
+                    {"event": "purchase", "timestamp": "2020-01-03", "properties": {ff_property: "control"}},
+                ],
+                "user_test_1": [
+                    {"event": "$pageview", "timestamp": "2020-01-02", "properties": {ff_property: "test"}},
+                    {"event": "purchase", "timestamp": "2020-01-03", "properties": {ff_property: "test"}},
+                ],
+                "user_test_2": [
+                    {"event": "$pageview", "timestamp": "2020-01-02", "properties": {ff_property: "test"}},
+                    {"event": "purchase", "timestamp": "2020-01-03", "properties": {ff_property: "test"}},
+                ],
+                "user_test_3": [
+                    {"event": "$pageview", "timestamp": "2020-01-02", "properties": {ff_property: "test"}},
+                ],
+                "user_test_4": [
+                    {"event": "$pageview", "timestamp": "2020-01-02", "properties": {ff_property: "test"}},
+                    {"event": "purchase", "timestamp": "2020-01-03", "properties": {ff_property: "test"}},
+                ],
+            },
+            self.team,
+        )
+
+        flush_persons_and_events()
+
+        query_runner = ExperimentFunnelQueryRunner(
+            query=ExperimentFunnelQuery(**experiment.metrics[0]["query"]), team=self.team
+        )
+        result = query_runner.calculate()
+
+        self.assertEqual(len(result.variants), 2)
+        for variant in result.variants:
+            self.assertIn(variant.key, ["control", "test"])
+
+        control_variant = next(v for v in result.variants if v.key == "control")
+        test_variant = next(v for v in result.variants if v.key == "test")
+
+        self.assertEqual(control_variant.success_count, 2)
+        self.assertEqual(control_variant.failure_count, 1)
+        self.assertEqual(test_variant.success_count, 3)
+        self.assertEqual(test_variant.failure_count, 1)
+
+        self.assertAlmostEqual(result.probability["control"], 0.407, places=2)
+        self.assertAlmostEqual(result.probability["test"], 0.593, places=2)
+
+        self.assertAlmostEqual(result.credible_intervals["control"][0], 0.1941, places=3)
+        self.assertAlmostEqual(result.credible_intervals["control"][1], 0.9324, places=3)
+        self.assertAlmostEqual(result.credible_intervals["test"][0], 0.2836, places=3)
+        self.assertAlmostEqual(result.credible_intervals["test"][1], 0.9473, places=3)
```

Comment:
i'm surprised this is not flaky, old runner needed the flaky decorator on these tests because sometimes the values will be a bit off.